lib/manager/cocoapods/readme.md

```diff
@@ -0,0 +1,238 @@
+## Overview
+
+#### Name of package manager
+
+Cocoapods
+
+---
+
+#### What language does this support?
+
+Swift, Objective-C
+
+---
+
+#### Does that language have other (competing?) package managers?
+
+Carthage, Swift Package Manager
+
+## Package File Detection
+
+#### What type of package files and names does it use?
+
+Podfile
+
+---
+
+#### What [fileMatch](https://renovatebot.com/docs/configuration-options/#filematch) pattern(s) should be used?
+
+`['(^|/)Podfile$']`
+
+---
+
+#### Is it likely that many users would need to extend this pattern for custom file names?
+
+No
+
+---
+
+#### Is the fileMatch pattern likely to get many "false hits" for files that have nothing to do with package management?
+
+Unlikely
+
+## Parsing and Extraction
+
+#### Can package files have "local" links to each other that need to be resolved?
+
+No
```

Comment:
So these local types of links don't point to another "Podfile", they point to something different? Presumably this path could also exist within the same repo?

```
pod 'Alamofire', :path => '~/Documents/Alamofire'
```